README.md

```diff
@@ -1,73 +1,36 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+Hell-Microverse-World
 
-> Description the project.
+This is my first project as a student at Microverse. It is a simple project built in HTML and CSS and it includes linters.
 
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
```

Comment:
Please list down the languages you have used to develop this project. Makes easy for other developers and employers to follow the code.